### PR TITLE
Campfire serve and open-browser

### PR DESCRIPTION
**Pull request by ten3roberts**
Improves the web workflow through campfire

### CODE REVIEW

**Review by philpax — APPROVED** [review]
lgtm, hope it all works!

File: campfire/src/web/serve.rs [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    walkdir::WalkDir::new(dir).into_iter().filter_entry(|v| {
+        let path = v.path();
+        let fname = v.file_name();
+
+        // if  path.starts_with(".") {
+        //     log::debug!("Ignoring hidden path: {path:?}");
+        //     return false;
+        // }
+
+        match fname.to_str() {
```
Comment (philpax) [inline campfire/src/web/serve.rs:240]: looking at this - maybe, as a future extension, it should respect `.gitignore`? definitely not necessary now, just figured I'd plant the idea

File: crates/model_import/src/lib.rs [hunk 2]
```diff
@@ -323,7 +323,7 @@ fn material_filter_matches(filter: &MaterialFilter, mat: &PbrMaterialDesc) -> bo
 //     }
 // }
 
-pub const MODEL_EXTENSIONS: &[&str] = &["glb", "fbx", "obj", "x"];
+pub const MODEL_EXTENSIONS: &[&str] = &["glb", "gltf", "fbx", "obj", "x"];
```
[the remark this answers is not in the document]
Comment (ten3roberts) [reply crates/model_import/src/lib.rs:326]: Yes, did it while pairing with @chaosprint and it just came up and we did it on the go while discussing how my edit-build-test workflow loop looks 😆

File: campfire/src/web/serve.rs (hunk 1 above)
Comment (ten3roberts) [reply campfire/src/web/serve.rs:240]: Yes, figured as such as well.

*Oh `.DS_STORE` how I've missed you*

**Comment by ten3roberts** [comment]
Accidentally fixes #644

**Pull request merged**